public/push-messaging-sw.js

```diff
@@ -0,0 +1,30 @@
+/* global importScripts firebase FCM_SENDER_ID */
+// Give the service worker access to Firebase Messaging.
+// Note that you can only use Firebase Messaging here, other Firebase libraries
+// are not available in the service worker.
+
+importScripts('/lib/firebase/firebase-app.js');
+importScripts('/lib/firebase/firebase-messaging.js');
+importScripts('/config/sw.js');
```

Comment:
Ah, I liked them in `index.html`. This way would be 1 extra http request too, but could load the config into a standard `TrustrootsConfig` object or something. I'm easy.